bigtable/google/cloud/bigtable/instance.py

```diff
@@ -311,6 +316,53 @@ def delete(self):
         """
         self._client.instance_admin_client.delete_instance(name=self.name)
 
+    def cluster(self, cluster_id, location_id=None, state=None,
+                serve_nodes=None, default_storage_type=None):
+        """Factory to create a cluster associated with this instance.
+
+        :type cluster_id: str
+        :param cluster_id: The ID of the cluster.
+
+        :type instance: :class:`~google.cloud.bigtable.instance.Instance`
+        :param instance: The instance where the cluster resides.
+
+        :type location_id: str
+        :param location_id: (Creation Only) The location where this cluster's
+                            nodes and storage reside. For best performance,
+                            clients should be located as close as possible to
+                            this cluster.
+                            For list of supported locations refer to
+                            https://cloud.google.com/bigtable/docs/locations
+
+        :type state: int
+        :param state: (`OutputOnly`)
```

Comment:
We're planning a new `list_clusters` method that will use states.  I'm not sure that `state` should belong here.  `list_clusters` can use the Cluster constructor.  This method is user facing and should only have properties that a user sets.

Reply:
Please remove the `state` parameter from the comments.